marklogic-data-hub/gradle.properties

```diff
@@ -59,7 +59,9 @@ mlFlowDeveloperRole=flow-developer-role
 mlFlowDeveloperUserName=flow-developer
 mlFlowDeveloperPassword=password
 mlDataHubAdminRole=data-hub-admin-role
+mlDataHubOperatorRole=data-hub-operator
+mlDataHubDeveloperRole=data-hub-developer
 
-mlModulePermissions=rest-reader,read,rest-writer,insert,rest-writer,update,rest-extension-user,execute,flow-developer-role,read,flow-developer-role,execute,flow-developer-role,insert,flow-operator-role,read,flow-operator-role,execute
+mlModulePermissions=rest-reader,read,rest-writer,insert,rest-writer,update,rest-extension-user,execute,flow-developer-role,read,flow-developer-role,execute,flow-developer-role,insert,flow-operator-role,read,flow-operator-role,execute,data-hub-developer,read,data-hub-developer,execute,data-hub-developer,insert,data-hub-operator,read,data-hub-operator,execute
```

Comment:
There's a separate story for modifying mlModulePermissions - it actually specifies setting the value to just the new data-hub roles and not using any of the existing ones.